src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/PublishTaxonomyFunction.cs

```diff
@@ -13,7 +13,9 @@ public class PublishTaxonomyFunction(
 {
     [Function("PublishTaxonomy")]
     public async Task PublishTaxonomy(
+#pragma warning disable IDE0060
         [QueueTrigger(PublishTaxonomyQueue)] PublishTaxonomyMessage message,
+#pragma warning restore IDE0060
```

Comment:
I see the `message` is unused. Is there a reason we cannot remove it?

If it can and you're avoiding it, don't feel the need to remove it on my behalf.

Reply:
You need to have some parameter to add the `QueueTrigger` attribute to.

You also can't use `PublishTaxonomyMessage _` otherwise the following startup error occurs:

`The 'PublishTaxonomy' function is in error: The binding name _ is invalid. Please assign a valid name to the binding.`